autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -135,7 +138,9 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleState(Cluster clus
 
     private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Cluster cluster,
             DistroXAutoscaleClusterRequest autoscaleClusterRequest) {
-
+        if (autoscaleClusterRequest.getUseStopStartMechanism() == null) {
+            autoscaleClusterRequest.setUseStopStartMechanism(asClusterCommonService.shouldEnableStopStartBasedOnEntitlement(cluster, autoscaleClusterRequest));
+        }
```

Comment:
Same here. Also, `AutoscaleClusterCommonService` has redundant code blocks. All such validations can be performed in `AlertValidator` instead.